src/components/ButtonToggle/ButtonToggle.types.ts

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+
+export type ButtonToggleSize = 'default' | 'big';
+export const buttonToggleSizes = ['default', 'big'];
+
+type BasicProps = {
+    /**
+     * Button label
+     */
+    label?: React.ReactNode;
```

Comment:
Must be string, isn't it ?
```suggestion
    label?: string;
```